aiidalab_widgets_base/structures.py

```diff
@@ -118,7 +118,7 @@ def _structure_importers(self, importers):
         """Preparing structure importers."""
         if not importers:
             raise ValueError("The parameter importers should contain a list (or tuple) of "
-                             "importers, got a falsy object.")
+                             "importers, got a false object.")
```

Comment:
Why? By "falsy" object I meant any of the following: `''`, `{}`, `False`, `[]`, `None`.